release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -1413,6 +1413,30 @@ sourceStrategy:
 
 * Previously, there was an eventual consistency issue in the AWS Terraform provider when updating new load balancers. Therefore, the installation would fail when trying to access the new load balancers. With this fix, the installer is now updated to an upstream Terraform provider, which ensures eventual consistency. As a result, the installation does not fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1898265[*BZ#1898265*])
 
+* Previously, the installation program had a list of required APIs that check for quota and permissions, and the list contained some unnecessary APIs that failed if the user did not provide the permissions. With this update, The list of APIs is split into `required` and `optional` where the `optional` APIs are not accessible. A warning message is displayed for the `optional` APIs. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2084280[*BZ#2084280*])
+
+* Previously, the `.apps` entry did not have the tag `kubernetes.io_cluster<infraID>` that was used by the installation program to delete code from the database to isolate all the resources created for a given cluster and delete them. With this update, a tag was added to the cluster ingress Operator during creation time which makes the entry visible to delete. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055601[*BZ#2055601*])
+
+* Previosly, `openshift-install` would fail when using the internal publishing strategy. With this update, the `openshift-install` no longer fails. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2047670[*BZ#2047670*])
+
+* Previously, there was an eventual consistency issue in the AWS Terraform provider when updating to newly created Virtual Private Clouds (VPCs). Consequently, the installation program would fail in attempt to access VPCs. With this update, the installation program is updated to the upstream Terraform provider and the install does not fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2043080[*BZ#2043080*])
+
+* Previously, there was an eventual consistency issue in the AWS Terraform provider when updating to newly created network interfaces. Consequently, installs would fail to access the network interfaces. With this update, the Terraform provider is updated to accept eventual consistency and install does not fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2047741[*BZ#2047741*])
```

Comment:
```suggestion
* Previously, there was an eventual consistency issue in the AWS Terraform provider when updating to newly created network interfaces. Consequently, installs would fail to access the network interfaces. With this update, the Terraform provider is updated to accept eventual consistency and installation does not fail. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2047741[*BZ#2047741*])
```